pkgs/development/compilers/llvm/17/libcxx/default.nix

```diff
@@ -54,6 +54,14 @@ stdenv.mkDerivation rec {
       hash = "sha256-LNoPg1KCoP8RWxU/AzHR52f4Dww24I9BGQJedMhFxyQ=";
       relative = "libcxx";
     })
+  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
```

Comment:
Any reason not to apply this unconditionally?  That way people are more likely to notice if some change breaks it.